.github/workflows/build_and_deploy_ext2_img.yml

```diff
@@ -0,0 +1,133 @@
+name: Create and deploy ext2 image to pages
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+  # Lets you supply arguments when you run a github action manually.
+    inputs:
+      DOCKER_BUILD_CONTEXT:
+        type: string
+        description: 'Path or url to the docker build context.'
+        required: true
+        default: 'dockerfiles/example'
+  
+      IMAGE_SIZE:
+        type: string
+        description: 'Size of the base ext2 image. 1GB is the max size for pages, 2GB for Gh release.. Refer to the fallocate(1) manpage for multiplicative suffixes'
+        required: true
+        default: '950MB'
+
+      IMAGE_NAME:
+        type: string
+        description: 'Name of final image. Example final url: https://www.github.io/<owner>/<repositoryName>/<image_name>'
+        required: true
+        default: 'base_image.ext2'
+
+      DEPLOY_TO_PAGES:
+        type: boolean
+        description: 'Deploy to pages.'
+        required: true
+        default: true
+
+      GITHUB_RELEASE:
+        type: boolean
+        description: 'Upload github release.'
+        required: true
+        default: true
```

Comment:
Let's default this to false to be nice and not pollute many gigs of space if not requested